main.tf

```diff
@@ -100,12 +97,20 @@ resource "kubernetes_deployment" "tfc_cloud_agent" {
               value = env.value
             }
           }
+          dynamic "env" {
+            for_each = local.tfc_agent_data_dir
+            content {
+              name  = "TFC_AGENT_DATA_DIR"
+              value = var.tfc_agent_data_dir
+            }
+          }
+
           resources {
-            limits {
+            limits = {
```

Comment:
Is this for a newer version of the K8s provider? Can you update to explicitly require that version or above?

Reply:
this is since 2 months ago , kubernetes provider version 2.0 
https://registry.terraform.io/providers/hashicorp/kubernetes/latest/docs/resources/deployment

Reply:
i've updated the required version but now the readme lint keeps failing

Reply:
updated the requirement versions